.idea/assetWizardSettings.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="WizardSettings">
+    <option name="children">
+      <map>
+        <entry key="vectorWizard">
+          <value>
+            <PersistentState>
+              <option name="children">
+                <map>
+                  <entry key="vectorAssetStep">
+                    <value>
+                      <PersistentState>
+                        <option name="values">
+                          <map>
+                            <entry key="assetSourceType" value="FILE" />
+                            <entry key="outputName" value="ic_baseline_arrow_back_24px" />
+                            <entry key="sourceFile" value="$USER_HOME$/Downloads/baseline-arrow_back-24px.svg" />
```

Comment:
@akench What does this file do? It looks like it's getting an image asset from our (dev) filesystem but I don't know what that would be needed for.